### PR TITLE
docs: update i18n doc regarding aot compilation

### PR DESCRIPTION
**Pull request by mlichtblau**
Add missing lines to code example to allow using ng serve with custom i18n configurations.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

After just updating the configurations for build, the cli throws an error when you try ng serve --configuration=fr. It is unclear that you have to change the configurations for serve as well.

## What is the new behavior?
It is now clear what modifications have to be done to use ng serve --configuration=fr

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information
This is my first pull request. I'm happy to make changes if requested :)

### CODE REVIEW

**Comment by mlichtblau** [comment]
I signed it!

File: aio/content/guide/i18n.md [hunk 1]
```diff
@@ -683,28 +683,40 @@ You also need to instruct the AOT compiler to use your translation configuration
...
+    }
+  }
+},
+"serve": {
+  ...
+  "configurations": {
     ...
+    "fr": {
+      "browserTarget": "angular.io-example:build:fr"
+    }
```
Comment (brandonroberts) [inline aio/content/guide/i18n.md]: This should be more generic. How about `*project name*:build:fr` instead of `angular.io-example:build:fr`

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Comment by brandonroberts** [comment]
@mlichtblau Thanks for the PR! I l left you some feedback.

**Review by ocombe — APPROVED** [review]

File: aio/content/guide/i18n.md (hunk 1 above)
Comment (mlichtblau) [reply aio/content/guide/i18n.md]: Thanks for the feedback. I changed the name to be more generic.

**Review by brandonroberts — APPROVED** [review]

**Pull request closed without merging**